AUTHORS

```diff
@@ -20,7 +20,7 @@ Aleksandra Hernik <aleksandra.hernik@intel.com>
 Paweł Kochanek <pawel.b.kochanek@intel.com>
 Katarzyna Wasiuta <katarzyna.wasiuta@intel.com>
 Krzysztof Filipek <krzysztof.filipek@intel.com>
-Michał Biesek <michal.biesek@intel.com>
+Michał Biesek
```

Comment:
Leave it as it was. No email addresses were ever removed from this file. Now it would be hard to track down which ones are valid and which ones are not. I don't see a reason for removing email addresses here.